lib/util/alias_helper.dart

```diff
@@ -0,0 +1,54 @@
+class AliasHelper {
+  static const List<String> _LOWERCASE_AC_ACCOUNT_ALIAS = [
+    'john gerrard',
+  ];
+
+  static String transform(String value, {bool isArtistOrCurator = false}) {
```

Comment:
I think the `isArtistOrCurator` param is unnecessary. This function is only called by FFArtist or FFCurator object.

If you still want to use this function for both internal and external artists, curators, I think it should be `isFFArtistOrCurator`

Reply:
Because it is in AliasHelper, general class to get displayed alias, i want to make the logic as same as in Feral File Web. For Artist and Curator, i have an extension to call this method with `isArtistOrCurator` = true